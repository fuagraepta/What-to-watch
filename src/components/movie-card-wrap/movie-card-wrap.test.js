import React from 'react';
import renderer from 'react-test-renderer';

import MovieCardWrap from './movie-card-wrap.jsx';

const mockElement = <div>Hello World</div>;

describe(`The component is rendered correctly`, () => {
  it(`The Movie-card-wrap correctly renders without transferred prop isTranslateTop`, () => {
    const movieCardWrap = renderer
      .create(
          <MovieCardWrap>
            {mockElement}
          </MovieCardWrap>
      )
      .toJSON();

    expect(movieCardWrap).toMatchSnapshot();
  });

  it(`The Movie-card-wrap correctly renders with transferred prop isTranslateTop`, () => {
    const movieCardWrap = renderer
      .create(
          <MovieCardWrap
            isTranslateTop = {true}
          >
            {mockElement}
          </MovieCardWrap>
      )
      .toJSON();

    expect(movieCardWrap).toMatchSnapshot();
  });
});
