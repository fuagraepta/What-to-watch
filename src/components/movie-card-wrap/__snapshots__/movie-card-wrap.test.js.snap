// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly The Movie-card-wrap correctly renders with transferred prop isTranslateTop 1`] = `
<div
  className="movie-card__wrap movie-card__translate-top"
>
  <div>
    Hello World
  </div>
</div>
`;

exports[`The component is rendered correctly The Movie-card-wrap correctly renders without transferred prop isTranslateTop 1`] = `
<div
  className="movie-card__wrap "
>
  <div>
    Hello World
  </div>
</div>
`;
